packages/strapi-admin/admin/src/containers/PermissionsManager/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators, compose } from 'redux';
+import { LoadingIndicatorPage, UserProvider, request } from 'strapi-helper-plugin';
+import {
+  getUserPermissions,
+  getUserPermissionsError,
+  getUserPermissionsSucceeded,
+} from './actions';
+import makeSelectPermissionsManager from './selectors';
+
+const PermissionsManager = ({
+  children,
+  isLoading,
+  getUserPermissions,
+  getUserPermissionsError,
+  getUserPermissionsSucceeded,
+  userPermissions,
+}) => {
+  const fetchUserPermissions = async (resetState = false) => {
+    if (resetState) {
+      // Show a loader
+      getUserPermissions();
+    }
+
+    try {
+      const { data } = await request('/admin/users/me/permissions', { method: 'GET' });
+
+      getUserPermissionsSucceeded(data);
+    } catch (err) {
+      console.error(err);
+      getUserPermissionsError(err);
+    }
+  };
+
+  useEffect(() => {
+    fetchUserPermissions(true);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Can we use react-query for this fetch ?

Reply:
I don't think it's a good idea here. We don't want to use the cache system